src/components/Superfest/SuperfestPage/CTA/MissionCTA.style.ts

```diff
@@ -0,0 +1,82 @@
+import { sequel65, sequel85, sora, urbanist } from '@/fonts/fonts';
+import type { BoxProps, Breakpoint } from '@mui/material';
+import { Box, alpha, darken } from '@mui/material';
+import { styled } from '@mui/material/styles';
+import { IconButtonPrimary } from 'src/components/IconButton';
+
+export const MissionCtaContainer = styled(Box)<BoxProps>(({ theme }) => ({
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'center',
+  alignItems: 'center',
+  padding: theme.spacing(8),
+  gap: theme.spacing(1.5),
+  cursor: 'pointer',
+  overflow: 'hidden',
+  textAlign: 'center',
+  margin: theme.spacing(6, 0),
+  transition: 'background-color 250ms',
+  borderRadius: '16px',
+  backgroundColor: '#69d7ff',
+  '&:hover': {
+    cursor: 'pointer',
+    backgroundColor:
+      theme.palette.mode === 'light'
+        ? darken('#69d7ff', 0.02) //todo: add to theme
+        : alpha('#69d7ff', 0.16),
+  },
+  [theme.breakpoints.up('sm' as Breakpoint)]: {
+    gap: theme.spacing(4),
+    flexDirection: 'row',
+  },
+}));
+
+export const MissionCtaTitle = styled(Box)<BoxProps>(({ theme }) => ({
+  fontFamily: sora.style.fontFamily,
+  fontWeight: 700,
+  color: '#000000',
```

Comment:
This could be `theme.palette.black.main`

Reply:
yes.